### ISSUE TITLE
Release/mark 1.19?

### ISSUE THREAD

**Issue by Phoenix-Starlight** [Szum123321/badstdout#4]

**Comment by Szum123321** [comment]
Sorry, i don't understand the question. I uploaded 1.19 version to CF over a month ago

**Comment by Phoenix-Starlight** [comment]
Ok, mail didn't get through. It's on modrinth that I didn't see it.

**Comment by Phoenix-Starlight** [comment]
Oh, I checked modrinth, that's why. My bad!

**Comment by triphora** [comment]
Indeed, it still is not released on [Modrinth](https://modrinth.com/mod/badstdout), please release it there.
